src/schema/v2/me/interestsConnection.ts

```diff
@@ -0,0 +1,58 @@
+import { GraphQLFieldConfig, GraphQLInt } from "graphql"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
+import { ResolverContext } from "types/graphql"
+import { paginationResolver } from "../fields/pagination"
+import {
+  UserInterestConnection,
+  userInterestCategoryEnum,
+  userInterestInterestTypeEnum,
+} from "../userInterests"
+
+export const InterestsConnection: GraphQLFieldConfig<void, ResolverContext> = {
```

Comment:
I wonder if we should rename it to `userInterestsConnection` to keep the naming of the underlying data structure and to align with the naming of the mutations.